src/app/client/src/app/modules/shared/services/config/url.config.json

```diff
@@ -160,7 +160,8 @@
     },
     "ADMIN": {
       "USER_SEARCH": "user/v1/search",
-      "ORG_SEARCH": "org/v1/search",
+      "ORG_SEARCH": "org/v2/search",
+      "ORG_SEARCH_CACHED": "org/v1/search",
```

Comment:
Please reword this. Rename ORG_SEARCH_CACHED -> ORG_SEARCH and ORG_SEARCH -> ORG_EXT_SEARCH